File: Algorithms/Recursive Algorithms/C++/Fibonacci.cpp
#include <bits/stdc++.h> 

using namespace std;

int fib(int n){
    if(n==1)
        return 1;
    else if(n==0)
        return 0;    
    else{
        return fib(n-1)+fib(n-2);
    }    
}

int main()
{
	int t;
	cout << "Enter a number: ";
    cin >> t;
    cout << "[ ";
    for(int i = 0; i < t; i++){
        int n; 
        cout << fib(i) << ", ";
    }
    cout << "\b\b]";
    return 0;
}

File: Algorithms/Recursive Algorithms/Go/fibonacci.go
// Golang program to
// print the Fibonacci series using recursion

package main

import "fmt"

func printFibonacii(a int, b int, n int) {
	var sum int = 0
	if n > 0 {
		sum = a + b
		fmt.Printf("%d ", sum)
		a = b
		b = sum
		printFibonacii(a, b, n-1)
	}
}

func main() {
	var a, b, n int

	a = 0
	b = 1

	fmt.Printf("Enter total number of terms: ")
	fmt.Scanf("%d", &n)

	fmt.Printf("Fibonacii series is : ")
	fmt.Printf("%d\t%d ", a, b)

	printFibonacii(a, b, n-2)
	fmt.Printf("\n")
}